### PR TITLE
add(tutorials): push metrics and logs from instance

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+</Message>
+
+<Message type="important">
+ - Having the default configuration on your agents might lead to more of your resources’ metrics being sent, a high consumption and a high bill at the end of the month.
+ - Sending metrics and logs for Scaleway resources or personal data using an external path is a billable feature. In addition, any data that you push yourself is billed, even if you send data from Scaleway products. Refer to the product pricing for more information.
```
Comment (RoRoJ) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:30]: Maybe a link to the pricing page would be good?

**Review by RoRoJ — APPROVED** [review]

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx (hunk 1 above)
Comment (nerda-codes) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:30]:
```suggestion
 - Sending metrics and logs for Scaleway resources or personal data using an external path is a billable feature. In addition, any data that you push yourself is billed, even if you send data from Scaleway products. Refer to the [product pricing](https://www.scaleway.com/en/pricing/?tags=available,managedservices-observability-cockpit) for more information.
```

**Review by bene2k1 — APPROVED** [review]

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+2. Copy your Instance's public IP as you will need it in the next steps.
+3. Open a terminal and type the following command to connect to your Instance:
+
+            ```bash
+            ssh root@<your Instance public IP>
+            ```
+4. Open another terminal and type the following command to clone our workshop repository on your local machine:
+
+            ```bash
+            git clone git@gitlab.infra.online.net:observability-paas/workshop.git
```
Comment (blazarecki) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:45]: Client will not have access to this Gitlab.
You can move the project in the public github.

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            ```
+7. Access the `workshop` folder using the following command:
+
+            ```bash
+            cd wokshop
+            ```
+8. Install [Golang](https://go.dev/) on your Instance using the following commands:
+
+            ```bash
+            wget https://dl.google.com/go/go1.13.5.linux-amd64.tar.gz
```
Comment (blazarecki) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:65]: You can use go1.20.x

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+This page shows you how to push your metrics and logs from a [Scaleway Instance](/compute/instances/concepts/#instance).
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+</Message>
+
+<Message type="important">
```
Comment (blazarecki) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:28]: You can also link to https://www.scaleway.com/en/docs/observability/cockpit/reference-content/understanding-cockpit-usage/ ?

**Review by blazarecki — CHANGES_REQUESTED** [review]

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx (hunk 2 above)
Comment (nerda-codes) [reply tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:45]: Should we create a specific public github to use for these kinds of tutos that involve Obs?

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx (hunk 3 above)
Comment (nerda-codes) [inline tutorials/pushing-metrics-logs-from-scw-instance/index.mdx:65]:
```suggestion
            wget https://dl.google.com/go/go1.20.linux-amd64.tar.gz
```

File: tutorials/pushing-metrics-logs-from-scw-instance/index.mdx (hunk 2 above)
Comment (nerda-codes) [reply tutorials/pushing-metrics-logs-from-scw-instance/index.mdx]: I created a repository "tuto-cockpit-instance" in which I put the files

**Pull request merged**